src/Symfony/Bundle/FrameworkBundle/Controller/ControllerHelper.php

```diff
@@ -351,6 +353,14 @@ public function createFormBuilder(mixed $data = null, array $options = []): Form
         return $this->container->get('form.factory')->createBuilder(FormType::class, $data, $options);
     }
 
+    /**
```

Comment:
in this class the return type for the createFormBuilder method should also be adjusted

Reply:
you are talking about the createForm in the ControllerHelper? good catch will do in another PR.

Reply:
Already done https://github.com/symfony/symfony/pull/62569